docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,294 @@
+
+Scalability::
+I can easily scale my cluster as my needs change, without any downtime or disruption to my services.
+
+Customization::
+I can customize my cluster with the features and components I need to run my workloads.
+
+Security::
+The service is built with security in mind and includes features such as encrypted communication, role-based access control (RBAC), and more to help me protect my data and workloads.
+
+Technical support::
+I can receive technical support from Kubernetes experts, who can help me resolve any issues or questions I may have, according to my SLA.
+
+
+== Iteration: Cluster Maintainence
+
+As a user of the service I can perform the following actions:
+
+Scheduling::
+I can schedule maintenance windows for my cloud Kubernetes environment, during which updates, patches, or other changes can be applied automatically.
+
+Notifications::
+I can receive notifications in advance of any scheduled maintenance, allowing me to plan accordingly and to minimize any disruption to my applications.
+
+Automated Updates::
+The service will automatically apply updates and patches to my cloud Kubernetes environment, ensuring that it is always running the latest and most secure version of the software.
+
+Monitoring::
+I can monitor the status of my cloud Kubernetes environment during the maintenance window, to ensure that the maintenance is operating as expected.
+
+
+== Iteration: Cluster Upgrade
+
+As a user of the service I can perform the following actions:
+
+Upgrade::
+I can upgrade my cloud Kubernetes clusters minor version.
+
+As an engineer of the service I can perform the following actions:
+
+Upgrade::
+I can upgrade my cloud Kubernetes clusters minor version.
+
+
+== Iteration: Backup and Restore
+
+
+As a user of the service I can perform the following actions:
+
+Data backup::
+I can create backups of my cloud Kubernetes environment, including the configuration, state, and data of my applications.
+
+Scheduling::
+I can schedule backups to run automatically, at a time and frequency that suits my needs.
+
+Data retention::
+I can configure data retention policies to ensure that backups are kept for the desired length of time.
+
+Restore::
+In the event of a failure, I can quickly and easily restore my cloud Kubernetes environment to a previous state, including all configuration, state, and data.
+
+Testing::
+I can test restores in a development environment before performing a full restore, to ensure that my data and applications are fully recoverable.
+
+
+== Iteration: Monitoring and Alerting
+
+As a user of the service I can perform the following actions:
+
+* I can have a monitoring service that will help me keep track of the health and performance of my cloud Kubernetes environment.
+* I can enable alerting on pre-defined SLIs to VSHN so that VSHN can resolve incidents with priority.
```

Comment:
```suggestion
* I have alerting on pre-defined SLIs to VSHN so that VSHN can resolve incidents with priority.
```

Reply:
As discussed in the meeting we had, we agreed to keep its as it is and not change "can" to "have". so we leave it as it is right now.